Cargo.toml

```diff
@@ -45,3 +44,6 @@ tendermint-proto = { git = "https://github.com/informalsystems/tendermint-rs.git
 default = ["native"]
 native = ["dep:tokio", "dep:reqwest", "dep:jsonrpsee"]
 verifier = []
+
+[patch.'https://github.com/Sovereign-Labs/sovereign.git']
```

Comment:
I suppose this can be removed, as we merged everything into research branch

Reply:
Good catch!